src/main/java/handler/CommandsHandler.java

```diff
@@ -0,0 +1,98 @@
+package handler;
+
+import dukeexecpetions.DeadlineEmptyException;
+import dukeexecpetions.EventEmptyException;
+import dukeexecpetions.InvalidCmd;
+import items.Items;
+import msg.Msg;
+import msg.StdMsgs;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class represents how Sir Duke handles the input with the appropriate given
+ * userInput
+ */
+public class CommandsHandler {
+    public enum Commands {
+        BYE,
+        LIST,
+        UNMARK,
+        MARK,
+        TODO,
+        DEADLINE,
+        EVENT,
+        ADD,
+        DELETE,
+    }
+
+    /**
+     * Converts the UserInput to a command. If command does not exist or entries are missing, error is thrown.
+     *
+     * @param userInput
+     * @param items The task list Sir Duke stores
+     * @return True or False, True if bye command is passed, false otherwise
+     * @throws InvalidCmd Command is not recognised
+     * @throws EventEmptyException Field to or from is empty
+     * @throws DeadlineEmptyException Field by is empty
+     */
+    public static boolean handleCommand(String userInput, Items items)
```

Comment:
Could the method name be more specific?
```suggestion
    public static boolean executeCommand(String userInput, Items items)
```